### DISCUSSION TITLE
Shortcut to copy filename with full path

### DISCUSSION THREAD

**Discussion by youngchiron** [sumatrapdfreader/sumatrapdf#3861] [Ideas]
Sometimes I want to save the file name with full path. 
Now, to get the filename, for example, O:\zotero_data_attachment\book\Froissart - 1978 - Chronicles.epub, I have to Ctrl+D, copy the info into notepad, and select the filename. 
If there is a shortcut to copy filename with full path, it will be useful.

**Comment by kjk** [comment]
Yes. You can `Ctrl + K` to get command palette and pick `Copy File Path` command which copies the full path to clipboard.

You can add a keyboard shortcut as described in https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

This assumes latest 3.5.x version.

**Reply by youngchiron** [reply]
Oh, Good. Thanks. I'll try.

**Comment by youngchiron** [comment]
Additional question: 
I found Copy file path menu by Ctrl+K. It's OK. 
And I understood how to make shortcut. It's OK. 
However, I can't find Copy File Path command in the command list. 
Could you help me with it?  Thanks in advance.

I found it in the source code. Is CmdCopyFilePath right?

It worked perfectly. Thanks.

**Reply by kjk** [reply]
Yes, it's `CmdCopyFilePath`
